quick_start.rst

```diff
@@ -706,7 +706,7 @@ test your web applications by using Behat with Mink.
 
 .. _`the Composer documentation`: https://getcomposer.org/download/
 .. _`the download page`: https://github.com/Behat/Behat/releases
-.. _`Mink`: https://github.com/behat/mink
+.. _`Mink`: https://github.com/minkphp/Mink
 .. _`What's in a Story?`: http://blog.dannorth.net/whats-in-a-story/
 .. _`Cucumber`: http://cukes.info/
```

Comment:
while we're at it, cukes.info now redirects to https://cucumber.io

Reply:
Done